class.jetpack-data.php

```diff
@@ -99,16 +99,6 @@ public static function is_usable_domain( $domain, $extra = array() ) {
 			return true;
 		}
 
-		// Check the IP to make sure it's pingable.
-		$ip = gethostbyname( $domain );
-
-		// Doing this again as I was getting some false positives when gethostbyname() flaked out and returned the domain.
-		$ip = filter_var( $ip, FILTER_VALIDATE_IP, FILTER_FLAG_IPV4 ) ? $ip : gethostbyname( $ip );
-
-		if ( ! filter_var( $ip, FILTER_VALIDATE_IP, FILTER_FLAG_NO_PRIV_RANGE | FILTER_FLAG_NO_RES_RANGE | FILTER_FLAG_IPV4 ) && ! self::php_bug_66229_check( $ip ) ) {
```

Comment:
This whole idea makes sense to me from a theoretical point, I haven't tested it in practice, and I actually don't know how. It would be great if you could provide some tips for that.

Also, we were checking `self::php_bug_66229_check` here, I can see it's not used anywhere else, why don't we remove it as well?